sourcemash/frontend/templates/help.tpl

```diff
@@ -0,0 +1,65 @@
+<div class="help-fixed-action-btn">
+  <span class="btn-floating btn-large">?</span>
+</div>
+
+<div id="help-modal" class="modal">
+  <div class="modal-content section no-pad-bot">
+    <div class="container">
+      <div class="row">
+
+        <h2 class="col s12 light center header flow-text">Confused?</h2>
```

Comment:
Maybe just Sourcemash then. I don't like "Confused?" because it seems disparaging to the user.